From 59eaff62ecc31c4e0527d2a0b9e36c96453a8944 Mon Sep 17 00:00:00 2001
From: Dmitry Rogozhkin <dmitry.v.rogozhkin@intel.com>
Date: Fri, 21 Aug 2020 12:17:57 -0700
Subject: [PATCH 1/2] switch to setuptools

Setuptools substituite distutils and provide convenient bdist_wheel
command to create wheel binary distributions installable with pip.

Signed-off-by: Dmitry Rogozhkin <dmitry.v.rogozhkin@intel.com>
---
 python/setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python/setup.py b/python/setup.py
index e6c6bab..896a23a 100755
--- a/python/setup.py
+++ b/python/setup.py
@@ -9,7 +9,7 @@ as well as a set of tools that allows a user to train and test a custom VMAF mod
 """
 
 import os
-from distutils.core import setup
+from setuptools import setup
 from Cython.Build import cythonize
 import numpy
 
-- 
1.8.3.1

